item.rb

```diff
@@ -19,10 +21,10 @@ def can_be_archived?
     current_date - Date.parse(@publish_date).year > 10
   end
 
-  #    def label=(label)
-  #     @label = label
-  #     @label.items.push(self) unless @label.items.inculede?(self)
-  #   end
+  # def label=(label)
+  #  @label = label
+  #  @label.items.push(self) unless @label.items.inculede?(self)
+  # end
```

Comment:
Remove the comments from these lines and change the line 26 to:
`@label.add_item(self) unless @label.items.include?(self)`